### ISSUE TITLE
Overfitting Problem

### ISSUE THREAD

**Issue by Jiexuanz** [aigc-apps/VideoX-Fun#238]
I try to overfit a random weight model (Wan2.1-1.3B) on a small set videos, but it fails. The loss remains stable around 0.4 and does not decrease further. However, fine-tuning on the pretrained model is effective?

**Comment by bubbliiiing** [comment]
The loss remains stable around 0.4, which seems too high. Could you share your training shell with me?
